vscode-extensions/tls-sync-vscode-shared/src/withTimeout.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+export async function withTimeout<T>(
```

Comment:
Do we really not have something like this in node-core-library already?

Reply:
Moved to `Async.runWithTimeoutAsync`